### DISCUSSION TITLE
redom roadmap

### DISCUSSION THREAD

**Discussion by data-handler** [redom/redom#379] [General]
What are the future plans for RE:DOM?

**Comment by pakastin** [comment]
RE:DOM is in use for example in our digital signage system iDiD (https://idid.fi, both manager and players) and Aviamaps (https://aviamaps.com). RE:DOM is very thin layer in front of DOM, so there's not much to constantly work, because web APIs do not change that much. That's why RE:DOM is so fast, reliable and future proof.

Of course if there's any bugs we'll fix right away, or if there's good ideas we're open for discussion.

Did this answer to your question? 😛

**Reply by data-handler** [reply]
👍

**Comment by joona** [comment]
I have around 30 projects depending on RE:DOM, so it's going to stay maintained in the future for sure. It's stable and reliable, so I don't see a lot of things to improve myself.

However, where we could improve is the ecosystem around RE:DOM. I should try to find some time to open source some parts of my toolchain built on top of it. It might be useful for others as well.

**Reply by pakastin** [reply]
Agreed! 👍 Nice to hear you enjoy using RE:DOM, I use it for all my projects as well 😎
